### DISCUSSION TITLE
6BF6AEC5-6810-467C-A3E4-2C4B365C7770

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#114] [General]
# 6BF6AEC5-6810-467C-A3E4-2C4B365C7770

AB Pictoris is a K-type main-sequence star about 163.47 light-years away in the constellation Pictor. In 2003, a very massive super-jovian planet or brown dwarf was discovered by the European Southern Observatory (ESO) through direct imaging

https://celestia.mobi/addon?item=6BF6AEC5-6810-467C-A3E4-2C4B365C7770

**Comment by exl001** [comment]
it doesn't seem to have it's atmosphere 🫤
